doc/source/serve/architecture.md

```diff
@@ -89,5 +116,4 @@ often used to implement [model composition](serve-model-composition).
 
 Serve utilizes Ray’s [shared memory object store](plasma-store) and in process memory
 store. Small request objects are directly sent between actors via network
-call. Larger request objects (100KiB+) are written to a distributed shared
-memory store and the replica can read them via zero-copy read.
+call. Larger request objects (100KiB+) are written to the object store and the replica can read them via zero-copy read (TODO: doesn't this only apply to numpy arrays?).
```

Comment:
Doesn't zero-copy read only apply to numpy arrays?

Reply:
@simon-mo do you know how this works?  I thought only numpy arrays had zero-copy read, not arbitrary Ray objects.